### ISSUE TITLE
Running 1070ti and 3070 

### ISSUE THREAD

**Issue by calebtho97** [trexminer/T-Rex#407]
Hi guys, im planning to run my 1070ti and 3070 together on trexminer. My 3070 will be arriving tomorrow. I was curious cause on my 1070ti Im using mt 4. However, I would like to set mt 6 and dag-build mode 2 on my rtx 3070 card. Is that possible?

Also, is a 700w 80plus bronze enough to run this 2 gpu?

Currently my 1070ti is at 35mh/s @134W.

Thank you!

**Comment by pdieppa** [comment]
I have not specified dag-build or mt settings and I get between 62.15 and 62.20 with a power limit of 125 watts. Idle, my 3070 draws 50 watts. These numbers are not at the wall since I have too many cards to figure out what each one draws. 

I run HiveOS and I have the Core at -600 and Memory at 2600 with a power limit at 120. I only get invalid shares when I mine Nicehash. 100% valids on my other pools. I use Nicehash few hours a day once a week because they pay in BTC, so I use that to pay for HiveOS.

You can specify the memory timings per card:

--mt                       Memory tweak mode (default: 0 - disabled). Range from 0 to 6. General recommendation
                                   is to start with 1, and then increase only if the GPU is stable.
                                   The effect is similar to that of ETHlargementPill.
                                   Supported on Pascal GPUs with GDDR5 or GDDR5X memory only.
                                   Requires running the miner with administrative privileges.
                                   Can be set to a comma separated list to apply different values to different cards.
                                   Example: --mt 4 (applies tweak mode #4 to all cards that support this functionality)
                                            --mt 3,3,3,0 (applies tweak mode #3 to all cards except the last one)

Same with --dag-build-mode

--dag-build-mode           [Ethash, ProgPOW, Octopus] Controls how DAG is built (default: 0).
                                   0 - auto (miner will choose the most appropriate mode based on the GPU model)
                                   1 - default (suitable for most graphics cards)
                                   2 - recommended for 30xx cards to prevent invalid shares
                                   Can be set to a comma separated list to apply different values to different cards.
                                   (eg: --dag-build-mode 1,1,2,1)

Good luck

**Comment by calebtho97** [comment]
Thank you so so so much!

**Comment by calebtho97** [comment]
50W, isnt that kinda low?

**Comment by pdieppa** [comment]
50 Watts idle (not mining). Actually it's way higher than my 5700 XTs. Those use between 7 and 9 watts idle. 
We always have to figure out the idle watts of all cards to make sure we don't have issues while the rig is booting up AND before the OC settings are applied. Idle wattage should always be way less than when they are mining but these 3070 can draw over 200 watts so, we always need to make sure the cards do not draw their max at anytime BEFORE the OC settings are applied or your power supply will not be happy.

**Comment by voyagerft** [comment]
So can you confirm that it is possible to make 1070ti and rtx 3070 work together? i thought it was not possible, at the moment i have a rig with 6 1070ti and 6 asus p104-100 cards .. can i add 6 more rtx 3070/90 cards?

**Comment by calebtho97** [comment]
Yep its 100% working. Both my graphic card are working in the same desktop. also, using the ddu to clean up all nvidia drivers. install 3070 drivers and the 1070ti is working like it should be. 

Currently, 
1070ti - 34mh/s
3070- 63.74mh/s
